admin/tool/policy/classes/api.php

```diff
@@ -331,6 +331,22 @@ public static function inactivate($policyid) {
         $DB->set_field('tool_policy', 'currentversionid', null, ['id' => $policyid]);
     }
 
+    /**
+     * Check if the user is a digital minor.
+     *
+     * @param int $dateofbirth
+     * @param string $country
+     * @return bool
+     */
+    public static function is_minor($dateofbirth, $country) {
+
+        $age = static::return_age($dateofbirth);
+        $agedigitalconsentmap = static::parse_age_digital_consent_map();
+
+        return array_key_exists($country, $agedigitalconsentmap) ?
+            $age < $agedigitalconsentmap[$country] : $age < $agedigitalconsentmap['*'];
+    }
+
```

Comment:
We will need some better handling of the situation where the map (by mistake) does not contain the wilcard `*` record. Also, may I ask to always add unit tests for all new / changed API methods. Thanks.

Reply:
Yes, I agree. Also, since the agedigitalconsentmap is stored as a string, we need to create a set of rules how the format should look like, so it would be easier to parse.